build/scripts/windows-sdk.ps1

```diff
@@ -120,6 +120,21 @@ elseif ($SdkVersion -eq "10.0.22000")
         exit $sdkExitCode
     }
 }
+elseif ($SdkVersion -eq "10.0.26100")
```

Comment:
are lines 108-122 cruft now?  (remove)

Reply:
Probably. Arguably the whole file should be and we switch to leveraging DevCheck

I'm focused on urgently unblocking the blocker so this cleanup should be in a later PR